src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const hasPartner =
+    person.sex === 'f' ? (
+      <p className="Person__partner">{person.partnerName} is my husband</p>
+    ) : (
+      <p className="Person__partner">{person.partnerName} is my wife</p>
+    );
+
```

Comment:
add also destructing 

`const { name, age, //and others } = person;`

Reply:
Good morning Volodymyr!!! Added destructing.